stripe/src/main/java/com/stripe/android/paymentsheet/model/ViewState.kt

```diff
@@ -1,16 +1,44 @@
 package com.stripe.android.paymentsheet.model
 
 import com.stripe.android.PaymentIntentResult
+import com.stripe.android.paymentsheet.PaymentOptionResult
 
-sealed class ViewState {
-    data class Ready(
-        val amount: Long,
-        val currencyCode: String
-    ) : ViewState()
+internal sealed class ViewState {
 
-    object Confirming : ViewState()
+    abstract fun isReady() : Boolean
```

Comment:
nit: `abstract fun isReady: Boolean` (more Kotliny)

Reply:
The parens are required.

Reply:
Oops, meant to write `abstract val isReady: Boolean`